lib/iex/lib/iex/info.ex

```diff
@@ -45,10 +45,9 @@ defimpl IEx.Info, for: Atom do
 
   defp info_module(mod) do
     extra =
-      if Code.get_docs(mod, :moduledoc) do
-        "Use h(#{inspect(mod)}) to access its documentation.\n"
-      else
-        ""
+      case Code.fetch_docs(mod) do
+        {:docs_v1, _, _, _, %{}, _, _} -> "Use h(#{inspect(mod)}) to access its documentation.\n"
```

Comment:
Maybe here the opposite would be better?

```elixir
{:error, _} -> ""
_ -> "Use h(#{inspect(mod)}) to access its documentation.\n"
```

This way if we add new versions in the future we won't need to change it.

Reply:
The module doc elem can also be `:none` or `:hidden` in which case `h/1` won't be able to return any documentation.

Reply:
Oh, I see, thank you.